shpc/utils/__init__.py

```diff
@@ -15,7 +15,7 @@
     get_tmpfile,
     mkdir_p,
     mkdirp,
-    rmdir_to_base,
+    rm_to_base,
```

Comment:
just a nit - could we name this `remove_to_base` so it's explicitly written out?

Reply:
or `delete_to_base` works to or `delete_to_root`